server/grpc/rpc.go

```diff
@@ -220,24 +220,25 @@ func (s *RPC) Init(c context.Context, id string, state rpc.State) error {
 		s.pubsub.Publish(message)
 	}()
 
-	workflow, err = pipeline.UpdateWorkflowToStatusStarted(s.store, *workflow, state)
+	workflow, err = pipeline.UpdateWorkflowStatusToRunning(s.store, *workflow, state)
 	if err != nil {
 		return err
 	}
 	s.updateForgeStatus(c, repo, currentPipeline, workflow)
+
 	return nil
 }
 
-// Done implements the rpc.Done function.
-func (s *RPC) Done(c context.Context, id string, state rpc.State) error {
-	workflowID, err := strconv.ParseInt(id, 10, 64)
+// Done marks the workflow with the given ID as done.
+func (s *RPC) Done(c context.Context, _workflowID string, state rpc.WorkflowState) error {
```

Comment:
please do not use param/variable names with underscore, would be better to use `workflowID` here and `id, err := strconv...` line below

Reply:
Wanted to not use `id` to avoid confusion with step.ID or pipeline.ID. Maybe `strWorkflowID`? Whats the issue with leading `_`s?

Reply:
it's not really how naming is usually done in Go, there shouldn't really be confusion if function param name would be named workflowID and in function body just use id as that variable is used only once and are 2 lines apart from converting from string to number so there shouldn't really be any confusion what it represents

Reply:
changed the vars to not start with `_`